### ISSUE TITLE
Select from broken in GAVcoin

### ISSUE THREAD

**Issue by jacogr** [ethcore/parity#2899]
![image](https://cloud.githubusercontent.com/assets/1424473/19735656/c95d5580-9bac-11e6-93eb-1402e1725ca9.png)

**Comment by jacogr** [comment]
As reported by @General-Beck (verified as broken), suspecting today's updates along the way

**Comment by jacogr** [comment]
Don't think it was from today, I think I missed this in the initial remove-personal-refactor - since there is a PR to re-instate that (https://github.com/ethcore/parity/pull/2879), I would keep this back slightly.

**Comment by ngotchac** [comment]
Can confirm, this is because `accounts` doesn't have a UUID anymore (without personal).
Could quick fix in the meanwhile ?
```javascript
//if (account.uuid) {
if (account.address) {
  return isPositive(account[options.gavBalance ? 'gavBalance' : 'ethBalance']);
}
```

**Comment by jacogr** [comment]
Closing due to https://github.com/ethcore/parity/pull/2879 reverting the personal_accountsInfo changes and verified to not have this issue. Can re-open if that is not merged for 1.4.
